ui/pages/settings/settings.container.js

```diff
@@ -115,6 +129,9 @@ const mapStateToProps = (state, ownProps) => {
     mostRecentOverviewPage: getMostRecentOverviewPage(state),
     pathnameI18nKey,
     remoteFeatureFlags,
+    settingsPageSnapsIds,
+    snapNameGetter,
```

Comment:
Wondering if we should just pass the name and IDs together so we don't need to pass the getter?